src/pages/en/core-concepts/project-structure.md

```diff
@@ -85,19 +85,19 @@ It is a common convention to store your CSS or Sass files in a `src/styles` dire
 
 ### `public/`
 
-The `public/` directory is for files and assets that do not need to be processed during Astro's build process. These files will be copied into the build folder untouched.
+The `public/` directory is for files and assets which shouldn't be processed during Astro's build process. These files will be copied into the build folder untouched.
```

Comment:
I prefer the original! 

You could say "that you do not want processed during Astro's build", but I think that's equally as good as the original. 

"Should" can be problematic in English. This sentence COULD actually mean... they SHOULDN'T get processed, but you never know what can happen..."  :) So, I'd prefer to keep it very clear. These are files that don't need processing, or these are files that you don't want processed.

Reply:
Ohh, I see